src/main/java/io/vlingo/http/resource/ErrorHandler.java

```diff
@@ -0,0 +1,12 @@
+package io.vlingo.http.resource;
+
+import io.vlingo.common.Completes;
+import io.vlingo.http.Response;
+
+public interface ErrorHandler {
```

Comment:
Hi Aleix, the implementation of the onError () can decide how to map the exception to the correct response. I could provide a base implementation of this wherein a exception type maps to a string error message. As is, this is the job of the implementer, but in any case the Response contains both the status code and the "entity"  or body content of the response (along with  headers, etc). 

Happy to write a generic one that is general purpose, since it will probably be useful to others.  

There is a second question of making the error handler as an optional of ResouceBuilder.resource(): it's kind of inconvenient to have to add it to each ResourceBuilder.

Reply:
> There is a second question of making the error handler as an optional of ResouceBuilder.resource(): it's kind of inconvenient to have to add it to each ResourceBuilder.

Yes... A developer might want to define it once and use it for each `ResourceHandler`. I won't wait for have this implementation to merge this PR, we can do the improvement in another PR ^^

Reply:
> Hi Aleix, the implementation of the onError () can decide how to map the exception to the correct response. I could provide a base implementation of this wherein a exception type maps to a string error message.

True. No need of a base implementation I guess.

Reply:
Sounds good.